src/pattern-library/components/patterns/prototype/LMSContentSelectionPage.tsx

```diff
@@ -0,0 +1,758 @@
+import classnames from 'classnames';
+import type { ComponentChildren } from 'preact';
+
+import {
+  ArrowRightIcon,
+  Button,
+  ButtonBase,
+  Card,
+  CardHeader,
+  CardContent,
+  GoogleDriveIcon,
+  IconButton,
+  InputGroup,
+  Input,
+  LinkIcon,
+  OneDriveIcon,
+  VitalsourceIcon,
+} from '../../../../';
+import type { IconComponent } from '../../../../types';
+import Library from '../../Library';
+
+function Button1({ children }: { children: ComponentChildren }) {
+  return (
+    <ButtonBase classes="bg-slate-0 p-2 rounded-sm border border-slate-3 gap-x-1">
+      {children}
+    </ButtonBase>
+  );
+}
+
+function Button2({
+  children,
+  contentType = 'pdf',
+}: {
+  children: ComponentChildren;
+  contentType?: string;
+}) {
+  return (
+    <ButtonBase classes="w-[15em] bg-slate-0 rounded-sm border border-slate-3 gap-x-2 items-center">
+      <div className="grow text-start p-2">
+        <strong>{children}</strong>
+      </div>
+      <div className="text-end p-2">
+        <span className="uppercase text-[11px]">{contentType}</span>
+      </div>
+    </ButtonBase>
+  );
+}
+
+function Button3({
+  children,
+  contentType = 'pdf',
+}: {
+  children: ComponentChildren;
+  contentType?: string;
+}) {
+  return (
+    <ButtonBase classes="w-[15em] bg-stone-50 rounded-sm border border-stone-300 gap-x-2 items-center">
+      <div className="grow text-start p-2">
+        <strong className="text-slate-600">{children}</strong>
+      </div>
+      <div className="text-end p-2">
+        <span className="uppercase text-[11px] text-stone-500">
+          {contentType}
+        </span>
+      </div>
+    </ButtonBase>
+  );
+}
+
+function Button4({
+  children,
+  contentType = 'pdf',
+}: {
+  children: ComponentChildren;
+  contentType?: string;
+}) {
+  return (
+    <ButtonBase classes="w-full bg-stone-50 rounded-sm border border-stone-300 gap-x-2 items-center">
+      <div className="grow text-start p-2">
+        <strong className="text-slate-600">{children}</strong>
+      </div>
+      <div className="text-end p-2">
+        <span className="uppercase text-[11px] text-stone-500">
+          {contentType}
+        </span>
+      </div>
+    </ButtonBase>
+  );
+}
+
+function Button5({
+  children,
+  icon,
+  contentType = 'pdf',
+}: {
+  children: ComponentChildren;
+  contentType?: string;
+  icon?: IconComponent;
+}) {
+  const Icon = icon ?? GoogleDriveIcon;
```

Comment:
You could use our standard PDF icon as the fallback icon, since that is the only content type supported by the sources that don't currently have their own icon.

Reply:
That's a great point. I will do that now.